jingo/__init__.py

```diff
@@ -49,7 +50,19 @@ def get_env():
     # TEMPLATE_DIRS and packages in INSTALLED_APPS.
     x = ((jinja2.FileSystemLoader, settings.TEMPLATE_DIRS),
          (jinja2.PackageLoader, settings.INSTALLED_APPS))
-    loaders = [loader(p) for loader, places in x for p in places]
+    loaders = []
```

Comment:
This should use the recommended way to iterate over the list of app config instances, `for app_config in apps.get_app_configs()`. So just pass in `apps.get_app_configs` as the option to the `PackageLoader` if Django's version is equal or higher than 1.7. Then iterate over the list of app configs and get their path attribute for the dotted import path.